sw/tools/parrot/bebop.py

```diff
@@ -125,8 +125,8 @@ def bebop_status():
     #check firmware version
     v = parrot_utils.check_version(tn, '').strip()
     print("Checking Bebop firmware version... " + v )
-    if v != '3.2.0':
-        print("Error: please upgrade your Bebop firmware to version 3.2.0!")
+    if v < '3.2.0':
```

Comment:
I am new to python. maybe some programming experts can make it better. I have tested with a simple python script and it just works. Some results are:
case1:
v=3.2.0
new_v=3.3.0
v>new_v = False
case2:
v=3.2.0
new_v=1.1.0
v>new_v = True
case3:
v=3.2.0
new_v=4.1.0
v>new_v = False